src/AWS.Deploy.Recipes/RecipeDefinitions/ASP.NETAppECSFargate.recipe

```diff
@@ -591,8 +629,7 @@
                     "Validators": [
                         {
                             "ValidatorType": "Range",
-                            "Configuration": 
-                            {
+                            "Configuration": {
```

Comment:
nit: can you remove whitespace changes?

Reply:
This is just how VS auto formats when you make edits. I don't really want to fight with the format back and forth.